adagios/etc/sudoers.d/adagios

```diff
@@ -1,3 +1,5 @@
-Defaults:nagios    !requiretty
-nagios             ALL = (root) NOPASSWD: /etc/init.d/nagios
+Defaults:%nagios    !requiretty
```

Comment:
quick question,

This file is used for more distros than just rhel. Can you confirm that changing this to a group is not breaking backwards compatibility ? I.e. what groups just debian use ?